src/main/java/selenium/pages/BurgerMenu.java

```diff
@@ -0,0 +1,35 @@
+package selenium.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.Wait;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import selenium.Driver;
+
+import static java.time.Duration.ofSeconds;
+import static org.openqa.selenium.support.ui.ExpectedConditions.textToBePresentInElementLocated;
+
+public class BurgerMenu {
```

Comment:
you can extend this class from from base page too

Reply:
Done